docs/docs/scanner/misconfiguration/custom/index.md

```diff
@@ -120,21 +119,50 @@ Trivy supports extra fields in the `custom` section as described below.
+| custom.subtypes            | Cloud Resource Subtypes                                           | Refer to the section on subtypes below                  |              N/A              | :material-close: | :material-check: |     :material-close:            |
+| custom.recommended_actions | Any characters                                                    | Describing what the user should do to resolve the issue |              N/A              | :material-close: | :material-check: |     :material-check:            |
+| custom.input.selector.type | Any item(s) in [this list][source-types]                          | More information provided below                         |              N/A              | :material-close: | :material-check: |     :material-check:            |
+| related_resource           | Any characters                                                    | URL to related resources                                |              N/A              | :material-close: | :material-check: |     :material-check:            |
+
+#### custom.avd_id and custom.id
+
+The AVD_ID can be used to link the check to the Aqua Vulnerability Database (AVD) entry. For example, the `avd_id` `AVD-AWS-0176` is the ID of the check in the [AWS Vulnerability Database](https://avd.aquasec.com/). If you are [contributing your check to trivy-policies](../../../../community/contribute/checks/overview.md), you need to generate an ID using `make id` in the [trivy-policies](https://github.com/aquasecurity/trivy-policies) repository. The output of the command will provide you the next free IDs for the different providers in Trivy.
+
+The ID is based on the AVD_ID. For instanc,e if the `avd_id` is `AVD-AWS-0176`, the ID is `ID0176`.
+
+#### custom.provider
+
+The `provider` field references the [provider](https://github.com/aquasecurity/defsec/tree/master/pkg/providers) available in Trivy. This should be the same as the provider name in the `pkg/iac/providers` directory, e.g. `aws`. 
+
+#### custom.service
+
+The `service` is the name of the service by the provider that the check targets. This should be the same as the service name in the `pkg/iac/providers` directory ([Link](https://github.com/aquasecurity/trivy/tree/main/pkg/iac/providers)), e.g. `rds`.
+
+#### custom.input
+
+The `input` tells Trivy what inputs this check should be applied to. Cloud provider checks should always use the `selector` input, and should always use the `type` selector with `cloud`. Check targeting Kubernetes yaml can use `kubenetes`, RBAC can use `rbac`, and so on.
+
+#### Subtypes in the custom data
+
+Subtypes currently only need to be defined for cloud providers [as detailed in the documentation.](./selectors.md/#enabling-selectors-and-subtypes)
 
+#### Scan Result
```

Comment:
We inform the user about policy namespaces in [this](https://aquasecurity.github.io/trivy/v0.49/docs/scanner/misconfiguration/#pass-custom-policies) part of the docs but I wonder if we can tie that in here as well because without passing a custom namespace, the custom checks will not be evaluated (trivy by default only runs in it's own namespace, unless specified by the user).